### PR TITLE
1.11.2 cherry pick

### PR DESCRIPTION
**Pull request by mlaventure**
The following commits have been cherry-picked:

5ff18be6d92a50d40727e6e8729ce2074191c1fc (#23067) Fixes Issue #22992: docker commit failing.
e24f320dd1a05f03c4f6ad6ee028b12deae6c443 (#22922) Enable seccomp on ppc64le
bde0bcf2d8112615c3015ac49a481b5af2e08e23 (#23032) Fix directory walker error checking
6d17d5bf9e3df79ecca891c45423ced79a1f48d6 (#22918) Release memoryStore locks before filter/apply
a8dc0ba83a4da515d4a2b22a890fe8cdfa168b3d (#22836) Update port info on network connect/disconnect
966e9126bb7f1f77f70b85d29da72e998b7231a8 (#22561) Wait for discovery on container start error 
4a26a657994881caeefbffe6b62d45fed5c9284f (#22227) Bump Go version to 1.6.2 from 1.6.1 where it was used 
d84150ab382661dd50cfcb65b144c4a1523370da (#22168) Add refcounts to graphdrivers that use fsdiff

### CODE REVIEW

**Comment by mlaventure** [comment]
That should be all the cherry-picks, just need to wait for this to go green :)

File: daemon/graphdriver/devmapper/driver.go [hunk 1]
```diff
@@ -151,26 +153,35 @@ func (d *Driver) Remove(id string) error {
 // Get mounts a device with given id into the root filesystem
 func (d *Driver) Get(id, mountLabel string) (string, error) {
 	mp := path.Join(d.home, "mnt", id)
+	rootFs := path.Join(mp, "rootfs")
+	if count := d.ctr.Increment(mp); count > 1 {
```
Comment (crosbymichael) [inline daemon/graphdriver/devmapper/driver.go]: Because of the cherry pick this should be `id` not `mp`

**Comment by thaJeztah** [comment]
can you update the description to include the last two cherry picks as well?

**Comment by mlaventure** [comment]
will do

**Comment by mlaventure** [comment]
Fixed the last cherry-pick @crosbymichael

**Comment by thaJeztah** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**